src/protocol/requests/produce/v6/request.js

```diff
@@ -0,0 +1,38 @@
+const requestV3 = require('../v3/request')
```

Comment:
I don't think we should "skip" versions when we delegate to an older version, even if the versions in between are in turn also delegating, because then if we change something in an in between version that change will be lost. It might not matter in this specific case, without having looked at v4 and v5, but if we are consistent in always delegating v7 -> v6 -> v5 etc. there's no chance of that issue happening.

Reply:
Plus it saves the reviewer from having to go look in all those in-between versions to check that there isn't anything going on there that we're missing :P

Reply:
Sure, I can change the reference.